requests/__init__.py

```diff
@@ -56,7 +56,7 @@
 # See https://github.com/kennethreitz/requests/issues/3213 as well.
 import ssl
 
-if ssl.OPENSSL_VERSION_INFO < (1, 0, 1):
+if hasattr(ssl, 'OPENSSL_VERSION_INFO') and getattr(ssl, 'OPENSSL_VERSION_INFO') < (1, 0, 1):
```

Comment:
So this disables all the SSL niceties Requests and cryptography provide for someone on 2.6 and has installed `requests[security]`. I'm not sure how much the Requests maintainers care about that particular environment config, but as it stands it's currently supported (albeit with a strong warning from cryptography to update your Python version, so I'm not sure how much longer this will remain to be true.)